.env.template

```diff
@@ -1,14 +1,14 @@
-PINECONE_API_KEY=your-pinecone-api-key
-PINECONE_ENV=your-pinecone-region
-OPENAI_API_KEY=your-openai-api-key
-ELEVENLABS_API_KEY=your-elevenlabs-api-key
-SMART_LLM_MODEL="gpt-4"
-FAST_LLM_MODEL="gpt-3.5-turbo"
+PINECONE_API_KEY=
+PINECONE_ENV=
+OPENAI_API_KEY=
```

Comment:
The previous version was self-documenting, making it easier for people to fill. So I guess the premise of this PR is just wrong. @Torantulino ?